pvlib/iotools/sodapro.py

```diff
@@ -0,0 +1,361 @@
+        NASA SRTM database
+    time_step: str, {'1min', '15min', '1h', '1d', '1M'}, default: '1h'
+        Time step of the time series, either 1 minute, 15 minute, hourly,
+        daily, or monthly.
+    time_reference: str, {'UT', 'TST'}, default: 'UT'
+        'UT' (universal time) or 'TST' (True Solar Time)
+    verbose: boolean, default: False
+        Verbose mode outputs additional parameters (aerosols). Only available
+        for 1 minute and universal time. See [1] for parameter description.
+    integrated: boolean, default False
+        Whether to return radiation parameters as integrated values (Wh/m^2)
+        or as average irradiance values (W/m^2) (pvlib preferred units)
+    label: {‘right’, ‘left’}, default: None
+        Which bin edge label to label time-step with. The default is ‘left’ for
+        all time steps except for ‘1M’ which has a default of ‘right’.
+    map_variables: bool, default: True
+        When true, renames columns of the DataFrame to pvlib variable names
+        where applicable. See variable CAMS_VARIABLE_MAP.
+    server: str, default: 'www.soda-is.com'
+        Main server (www.soda-is.com) or backup mirror server (pro.soda-is.com)
+
+    Returns
+    -------
+    data: pandas.DataFrame
+        Timeseries data, see Notes for columns
+    meta: dict
+        Metadata of the requested time-series
+
+    Notes
+    -----
+    In order to use the CAMS services, users must register for a free SoDa
+    account using an email address [1]_.
+
+    The returned data DataFrame includes the following fields:
+
+    ========================  ======  =========================================
+    Key, mapped key           Format  Description
+    ========================  ======  =========================================
+    **Mapped field names are returned when the map_variables argument is True**
+    ---------------------------------------------------------------------------
+    Observation period        str     Beginning/end of time period
+    TOA, ghi_extra            float   Horizontal radiation at top of atmosphere
+    Clear sky GHI, ghi_clear  float   Clear sky global radiation on horizontal
+    Clear sky BHI, bhi_clear  float   Clear sky beam radiation on horizontal
+    Clear sky DHI, dhi_clear  float   Clear sky diffuse radiation on horizontal
+    Clear sky BNI, dni_clear  float   Clear sky beam radiation normal to sun
+    GHI, ghi*                 float   Global horizontal radiation
+    BHI, bhi*                 float   Beam (direct) radiation on horizontal
+    DHI, dhi*                 float   Diffuse horizontal radiation
+    BNI, dni*                 float   Beam (direct) radiation normal to the sun
+    Reliability*              float   Reliable data fraction in summarization
+    ========================  ======  =========================================
+
+    *Parameters only returned if identifier='cams_radiation'. For description
```

Comment:
Sphinx is unhappily trying to interpret these asterisks in a way I don't understand.  Not sure what to do about it.  Maybe replace the asterisks with some other symbol like †, or maybe break out those parameters into their own table?

Reply:
As suggested I have used †, instead. It still shows up as an asterisk in the documentation built by readthedocs. Can you see if it still complains (or where can I see that)?

Reply:
I think the source code still uses the asterisk, maybe you didn't push that commit to github?

To see the sphinx build log, navigate to the list of RTD builds either through the version switcher at the bottom left of any RTD page, or directly with [this link](https://readthedocs.org/projects/pvlib-python/builds/).  Then choose the build you want, and on the right side of the page there is a "view raw" link that takes you to the raw log file.  Right now there's a ton of warnings about forecast.py and ModelChain that I think have already been resolved on `master`, so you may want to merge master into your branch just to get rid of those.

Reply:
The error warning was: "WARNING: Inline emphasis start-string without end-string.". 

In reStructuredText an asterisk is the start of italiced text, so Sphinx was probably looking for an ending asterisk. For future reference, an asterisk can be used inline, [but it has to be escaped with a backslash, i.e., \*](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html).